catalyst/core/callbacks/early_stop.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+from catalyst.core import _State, Callback, CallbackNode, CallbackOrder
+
+_NUM_LOADER_STEPS = int(os.environ.get("NUM_LOADER_STEPS", 2))
```

Comment:
Can you add `CHECK` or `CHECK_RUN` to env variable names, because they may be misleading overwise?

Reply:
fixed